### PR TITLE
Migrate to cli v2

### PR DESCRIPTION
**Pull request by gunjan01**
The recent changes in the cli package used in this repository has had some breaking changes and has broken the download of this project. Since we are not vendoring this dependency in `external admin service` and directly pull it in our Dockerfile, it has broken our builds.
This PR accomodates the necessary changes required to migrate to cli's version 2 apis.

### CODE REVIEW

**Review by manishrocksag — COMMENTED** [review]
Put a PR description and a proper reasons on why we are doing it. This will help us to keep track of it in the future.

File: cmd/valiant/cmd.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ func main() {
 	app.Name = "valiant"
 	app.Usage = "Validate request and responses to a web site."
 
-	app.Commands = []cli.Command{
+	app.Commands = []*cli.Command{
```
Comment (manishrocksag) [inline cmd/valiant/cmd.go:15]: Shouldnt we vendor cli ? If cli package is updated in future again then these changes will break again.

Comment (gunjan01) [reply cmd/valiant/cmd.go]: We can do that too. But we need to do it in external admin

Comment (gunjan01) [reply cmd/valiant/cmd.go:15]: I agree. That can be added as tech debt.

Comment (manishrocksag) [reply cmd/valiant/cmd.go:15]: External admin does not have a dependency on cli but valiant has. Its version should be fixed in valiant. We might want to use a different version of cli in external admin according to our needs. For now I guess it should be fine.

**Review by manishrocksag — APPROVED** [review]

**Review by sebdah — APPROVED** [review]

**Pull request merged**